cpp/tests/utilities/column_wrapper.cuh

```diff
@@ -336,6 +341,37 @@ struct column_wrapper {
     return true;
   }
 
+  /**---------------------------------------------------------------------------*
```

Comment:
Why not have the `operator==(column_wrapper<ColumnType> const& rhs)` call the `operator==(gdf_column const& rhs)` to save duplicating code?

Reply:
Great idea.